S7.Net/Types/Struct.cs

```diff
@@ -178,6 +189,32 @@ public static int GetStructSize(Type structType)
                         info.SetValue(structValue, LReal.FromByteArray(data));
                         numBytes += 8;
                         break;
+                    case "String":
+                        S7StringAttribute? attribute = info.GetCustomAttributes<S7StringAttribute>().SingleOrDefault();
+                        if (attribute == default(S7StringAttribute))
+                            throw new ArgumentException("Please add S7StringAttribute to the string field");
+
+                        numBytes = Math.Ceiling(numBytes);
+                        if ((numBytes / 2 - Math.Floor(numBytes / 2.0)) > 0)
+                            numBytes++;
+
+                        // hier auswerten
```

Comment:
Would prefer English comments only 😀

Reply:
Ups, just copied/inserted the code above without changing the comment xD